core/navigation.js

```diff
@@ -297,6 +313,10 @@ Blockly.Navigation.insertFromFlyout = function() {
   //TODO: IF this is null then we need to add to the correct place on the
```

Comment:
Can we remove this TODO since I think it is taken care of in insertBlockToWs?

Reply:
Done